daemon/entrypoint.sh

```diff
@@ -65,6 +65,11 @@ chown ceph. /var/run/ceph
 # based on the data store
 case "$KV_TYPE" in
    etcd|consul)
+      if [[ ! -e "/etc/ceph/populate.sh" ]]; then
+        source /populate.sh
```

Comment:
I'm a bit hesitant to just directly execute the populate script without any modifications.  Perhaps it is safe, but I seem to recall it not being suitable for general use, with various assumptions made which may not be good practice.